### PR TITLE
new package: pkcs11-helper 1.25.1

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/libpkcs11-helper/sources [hunk 1]
```diff
@@ -0,0 +1 @@
+https://github.com/OpenSC/pkcs11-helper/archive/pkcs11-helper-1.25.1.tar.gz
```
Comment (dylanaraps) [inline community/libpkcs11-helper/sources]: Try this source: `https://github.com/OpenSC/pkcs11-helper/releases/download/pkcs11-helper-1.25.1/pkcs11-helper-1.25.1.tar.bz2`. It should remove the autoreconf requirement.

**Comment by dylanaraps** [comment]
Rename this to `pkcs11-helper`. :+1:

**Comment by a-schaefers** [comment]
This and openvpn are ready now

File: community/pkcs11-helper/depends [hunk 2]
```diff
@@ -0,0 +1,2 @@
+make
```
Comment (dylanaraps) [inline community/pkcs11-helper/depends]: `make` can be removed from here.

**Comment by a-schaefers** [comment]
fixed

**Pull request merged**